### PR TITLE
WIP: Gnome 3.30

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

**Comment by Cogitri** [comment]
Seems like I can't build most stuff due to the icu update which is currently being done, so not everything has been tested yet...

**Comment by Cogitri** [comment]
Yup, am testing stuff, thanks for the notification

**Comment by Cogitri** [comment]
Needs https://github.com/void-linux/void-packages/pull/2346 for GJS

**Comment by Cogitri** [comment]
And https://github.com/void-linux/void-packages/pull/2363

**Comment by maxice8** [comment]
IMO this should be broken into various PRs, easier to review and actually runnable on Travis

**Comment by Cogitri** [comment]
Hm, okay, but some of the core stuff has to be bumped in one PR because it's so interconnected. I will open seperate PRs for the other applications.

File: srcpkgs/gdm/template [hunk 1]
```diff
@@ -19,9 +18,9 @@ checkdepends="check-devel"
 short_desc="GNOME Display Manager"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
 license="GPL-2.0-or-later"
-homepage="http://www.gnome.org"
+homepage="https://gitlab.gnome.org/GNOME/gdm"
```
Comment (maxice8) [inline srcpkgs/gdm/template]: I think this is more suitable: https://wiki.gnome.org/Projects/GDM

File: srcpkgs/mutter/template [hunk 2]
```diff
@@ -1,42 +1,31 @@
...
 makedepends="elogind-devel glib-devel gnome-desktop-devel json-glib-devel
- libSM-devel libXtst-devel libcanberra-devel libinput-devel
- startup-notification-devel upower-devel"
+ libglib-devel libSM-devel libXtst-devel libcanberra-devel libinput-devel
+ MesaLib-devel pipewire-devel startup-notification-devel wayland-protocols"
 depends="desktop-file-utils zenity"
 short_desc="Lightweight GTK+3 window manager"
 maintainer="Enno Boland <gottox@voidlinux.eu>"
-homepage="http://www.gnome.org"
+homepage="https://gitlab.gnome.org/GNOME/mutter"
```
Comment (maxice8) [inline srcpkgs/mutter/template]: https://wiki.gnome.org/Projects/Mutter/

File: srcpkgs/nautilus/template [hunk 3]
```diff
@@ -1,31 +1,27 @@
...
-makedepends="exempi-devel gnome-autoar-devel gnome-desktop-devel libexif-devel
- libgexiv2-devel libnotify-devel tracker-devel"
+makedepends="exempi-devel gnome-autoar-devel gnome-desktop-devel gtk+3-devel
+ libexif-devel libgexiv2-devel libglib-devel libnotify-devel libseccomp-devel
+ libX11-devel libxml2-devel tracker-devel"
 depends="desktop-file-utils gsettings-desktop-schemas hicolor-icon-theme"
 short_desc="The GNOME file manager"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 homepage="https://wiki.gnome.org/Nautilus"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
```
Comment (maxice8) [inline srcpkgs/nautilus/template]: please move homepage to directly below license=

File: srcpkgs/yelp-xsl/template [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
 makedepends="libxslt-devel"
 short_desc="Help browser for GNOME desktop - XSL and misc files"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2.0-or-later"
-homepage="http://www.gnome.org"
+homepage="https://wiki.gnome.org/Apps/Yelp/Xsl"
 distfiles="${GNOME_SITE}/$pkgname/${version%.*}/$pkgname-$version.tar.xz"
-checksum=8ccdf47b31acbdd26a7380b3bc6700742a7aff91a54dab279fd5ea8b43b53f93
+checksum=976c4436d2b0189ced004e52af6321bf9d00fe82187df0eb0e81ce732c3806cd
 noarch=yes
```
Comment (maxice8) [inline srcpkgs/yelp-xsl/template]: please move noarch=yes to directly below revision=

File: srcpkgs/evolution-data-server/template [hunk 5]
```diff
@@ -1,22 +1,22 @@
 # Template file for 'evolution-data-server'
 pkgname=evolution-data-server
-version=3.28.4
-revision=3
+version=3.30.0
+revision=1
 build_style=cmake
 configure_args="-DENABLE_GOOGLE_AUTH=OFF -DENABLE_UOA=OFF
```
Comment (maxice8) [inline srcpkgs/evolution-data-server/template]: Please remove -DENABLE_GOOGLE_AUTH as it is an unused option, see https://github.com/void-linux/void-packages/pull/2400

**Comment by maxice8** [comment]
Only package that remains is gjs which can't be merged because mozjs60 miscompiles on x86_64-musl

**Comment by Cogitri** [comment]
Closed due to everything but GJS being merged

**Pull request closed without merging**